Transient Dependency - Compile Issue (Typia Originating)

When I go to build a project with typia installed (not samchon/openAPI), I get this error when running the build command.

I have deleted the node modules folder, the package-lock and ran `npm install`. Issue persists.
Same with `npm ci`.

I think there may be a typing metadata issue that is causing the below error.

Versions used:
- Node.JS: `v22.15.0`
- Typia: `~9.3.0`
- TypeScript: `~5.8.3`


```powershell
npm run build:Prod

> my-project@3.0.0 build:Prod
> tsc --sourceMap false

node_modules/@samchon/openapi/lib/structures/IMcpLlmController.d.ts:82:20 - error TS2307: Cannot find module '@modelcontextprotocol/sdk/client/index.d.ts' or its corresponding type declarations.

82     client: import("@modelcontextprotocol/sdk/client/index.d.ts").Client;
                      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/@samchon/openapi/lib/structures/IMcpLlmController.d.ts:82
```


dependencies:
```JSON
{
    "@azure/arm-appservice": "~16.0.0",
    "@azure/arm-authorization": "~9.0.0",
    "@azure/arm-desktopvirtualization": "~1.2.0",
    "@azure/arm-keyvault": "~3.2.0",
    "@azure/arm-network": "~33.5.0",
    "@azure/arm-resources": "~6.0.0",
    "@azure/arm-storage": "~18.5.0",
    "@azure/identity": "~4.10.0",
    "@azure/keyvault-secrets": "~4.9.0",
    "@azure/storage-blob": "~12.27.0",
    "@microsoft/microsoft-graph-client": "~3.0.7",
    "@reduxjs/toolkit": "~2.8.2",
    "@shi-corp/objectid-converter": "~2.0.8",
    "@shi-corp/sdk-data-gateway": "~2.1.0",
    "compression": "~1.8.0",
    "express": "~5.1.0",
    "express-rate-limit": "~7.5.0",
    "helmet": "~8.1.0",
    "jws": "~4.0.0",
    "node-schedule": "~2.1.1",
    "ts-patch": "~3.3.0",
    "typia": "~9.3.0"
  }
```

devDependencies:
```JSON
{
    "@microsoft/microsoft-graph-types-beta": "~0.43.0-preview",
    "@stylistic/eslint-plugin": "~4.2.0",
    "@types/chai": "~5.2.2",
    "@types/compression": "~1.7.5",
    "@types/express": "~5.0.2",
    "@types/jws": "~3.2.10",
    "@types/mocha": "~10.0.10",
    "@types/node": "~22.15.19",
    "@types/node-schedule": "~2.1.7",
    "@types/swagger-ui-express": "~4.1.8",
    "chai": "~5.2.0",
    "esbuild-runner": "~2.2.2",
    "eslint": "~9.27.0",
    "eslint-plugin-jsdoc": "~50.6.17",
    "javascript-obfuscator": "~4.1.1",
    "mocha": "~11.4.0",
    "typescript": "~5.8.3",
    "typescript-eslint": "~8.32.1"
  }
```

--- a/build/post.ts
+++ b/build/post.ts
@@ -1,0 +1,22 @@
+import fs from "fs";
+
+const main = async (): Promise<void> => {
+  const input: string = await fs.promises.readFile(
+    `${__dirname}/../src/structures/IMcpLlmController.ts`,
+    "utf8",
+  );
+  await fs.promises.writeFile(
+    `${__dirname}/../lib/structures/IMcpLlmController.d.ts`,
+    input,
+    "utf8",
+  );
+  await fs.promises.writeFile(
+    `${__dirname}/../lib/structures/IMcpLlmController.d.mts`,
+    input,
+    "utf8",
+  );
+};
+main().catch((error) => {
+  console.error(error);
+  process.exit(-11);
+});

--- a/package.json
+++ b/package.json
@@ -1,13 +1,13 @@
 {
   "name": "@samchon/openapi",
-  "version": "4.3.0",
+  "version": "4.3.1",
   "description": "OpenAPI definitions and converters for 'typia' and 'nestia'.",
   "main": "./lib/index.js",
   "module": "./lib/index.mjs",
   "typings": "./lib/index.d.ts",
   "scripts": {
     "prepare": "ts-patch install",
-    "build": "rimraf lib && tsc && rollup -c",
+    "build": "rimraf lib && tsc && rollup -c && ts-node build/post.ts",
     "dev": "rimraf lib && tsc --watch",
     "typedoc": "typedoc --plugin typedoc-github-theme --theme typedoc-github-theme"
   },
@@ -40,20 +40,22 @@
   },
   "homepage": "https://samchon.github.io/openapi/api",
   "devDependencies": {
-    "@modelcontextprotocol/sdk": "^1.11.1",
+    "@modelcontextprotocol/sdk": "^1.11.4",
     "@rollup/plugin-terser": "^0.4.4",
     "@rollup/plugin-typescript": "^12.1.2",
     "@trivago/prettier-plugin-sort-imports": "^5.2.2",
+    "@types/node": "^20.12.7",
     "gh-pages": "^6.3.0",
     "prettier": "^3.5.3",
     "rimraf": "^5.0.5",
     "rollup": "^4.18.1",
     "rollup-plugin-auto-external": "^2.0.0",
     "tinyglobby": "^0.2.10",
+    "ts-node": "^10.9.2",
     "ts-patch": "^3.3.0",
     "typedoc": "^0.27.6",
     "typedoc-github-theme": "^0.2.1",
-    "typescript": "~5.7.2"
+    "typescript": "~5.8.3"
   },
   "sideEffects": false,
   "files": [

--- a/src/OpenApi.ts
+++ b/src/OpenApi.ts
@@ -962,11 +962,11 @@ export namespace OpenApi {
        *
        * If the value is `true`, it means that the additional properties are not
        * restricted. They can be any type. Otherwise, if the value is
-       * {@link IOpenAiSchema} type, it means that the additional properties must
+       * {@link IJsonSchema} type, it means that the additional properties must
        * follow the type schema info.
        *
        * - `true`: `Record<string, any>`
-       * - `IOpenAiSchema`: `Record<string, T>`
+       * - `IJsonSchema`: `Record<string, T>`
        */
       additionalProperties?: boolean | IJsonSchema;
 

